build/docker/debian.Dockerfile

```diff
@@ -81,5 +81,22 @@ RUN apt -y update && apt -y upgrade && apt -y install openjdk-11-jre \
 
 RUN dotnet --version
 
+RUN apt update
```

Comment:
either remove or bring in line with row 86

(there more RUNs you have in a Dockerfile, the more inefficient the end image becomes)

Reply:
if you do move it to row 86, remember to the same cleanup as you do for php installation (`apt -y clean && rm -rf /var/lib/apt/lists/*`)